src/MinecraftSkins.php

```diff
@@ -30,18 +29,27 @@ public static function head($rawSkin, $scale = 1, $vertRot = 0, $horRot = 0, $he
 
         return $canvas;
     }
+	
```

Comment:
Please add some basic documentation, if it's intended to be used publicly, otherwise set it to private.

Reply:
Added some clarifications and comments. I'll add the documentation as soon as I have more free time.